collect_app/src/main/AndroidManifest.xml

```diff
@@ -272,12 +276,12 @@ the specific language governing permissions and limitations under the License.
          -->
 
         <provider
-            android:name=".provider.FormsProvider"
-            android:authorities="org.bcss.collect.android.provider.odk.forms"
+            android:name="org.odk.collect.android.provider.FormsProvider"
+            android:authorities="org.odk.collect.android.provider.odk.forms"
             android:exported="true" />
         <provider
-            android:name=".provider.InstanceProvider"
-            android:authorities="org.bcss.collect.android.provider.odk.instances"
+            android:name="org.odk.collect.android.provider.InstanceProvider"
```

Comment:
We can't use  android:authorities="org.odk.collect.android.provider.odk.forms" because it clashes with odk
making it impossible to have both of the apps installed at the same time

Reply:
you would get 
`PackageManager: Package couldn't be installed in /data/app/org.bcss.collect.android-ZMq3S9l1kLH3N_b2-_Jiaw==
    com.android.server.pm.PackageManagerException: Can't install because provider name org.odk.collect.android.provider.odk.forms (in package org.bcss.collect.android) is already used by org.odk.collect.android
        at com.android.server.pm.PackageManagerService.assertPackageIsValid(PackageManagerService.java:11470)
        at com.android.server.pm.PackageManagerService.scanPackageDirtyLI(PackageManagerService.java:10726)
        at com.android.server.pm.PackageManagerService.scanPackageLI(PackageManagerService.java:10654)
        at com.android.server.pm.PackageManagerService.scanPackageTracedLI(PackageManagerService.java:10630)
        at com.android.server.pm.PackageManagerService.installNewPackageLIF(PackageManagerService.java:17585)
        at com.android.server.pm.PackageManagerService.installPackageLI(PackageManagerService.java:18849)
        at com.android.server.pm.PackageManagerService.installPackageTracedLI(PackageManagerService.java:18381)
        at com.android.server.pm.PackageManagerService.-wrap33(Unknown Source:0)
        at com.android.server.pm.PackageManagerService$7.run(PackageManagerService.java:15842)
        at android.os.Handler.handleCallback(Handler.java:790)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:164)
        at android.os.HandlerThread.run(HandlerThread.java:65)
        at com.android.server.ServiceThread.run(ServiceThread.java:46)`